ports/zeldoschallenge2/The Legend of Zelda - Zeldo's Challenge 2.sh

```diff
@@ -22,7 +18,7 @@ solarus_dir="$HOME/portmaster-solarus"
 solarus_file="$controlfolder/libs/${runtime}.squashfs"
 
 # Exports
-export LD_LIBRARY_PATH="$GAMEDIR/libs:$solarus_dir"
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir"
```

Comment:
Does this not need `$LD_LIBRARY_PATH` at the end for platforms like Crossmix?
```suggestion
export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$solarus_dir:$LD_LIBRARY_PATH"
```